parlai/mturk/core/dev/agents.py

```diff
@@ -14,13 +14,11 @@
 import parlai.mturk.core.dev.shared_utils as shared_utils
 
 # Special act messages for failure states
+# TODO replace with raised errors
 MTURK_DISCONNECT_MESSAGE = '[DISCONNECT]'  # Turker disconnected from conv
 TIMEOUT_MESSAGE = '[TIMEOUT]'  # the Turker did not respond but didn't return
 RETURN_MESSAGE = '[RETURNED]'  # the Turker returned the HIT
 
-# TODO move time management into another class, this way we can handle it
```

Comment:
This was actually already handled, agents don't manage their own heartbeats anymore